src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/ConsignmentRepository.scala

```diff
@@ -56,6 +56,11 @@ class ConsignmentRepository(db: Database, timeSource: TimeSource) {
     db.run(query.result)
   }
 
+  def getConsignments(): Future[Seq[ConsignmentRow]] = {
+    val query = Consignment
+    db.run(query.result)
```

Comment:
Sorry, I've just realised I'm probably looking at this PR too early. I'm guessing you'll be adding a sort order and cursor here?

Reply:
Yes, I've not actually implemented the pagination yet.